aviary/validation_cases/benchmark_tests/test_FLOPS_based_sizing_N3CC.py

```diff
@@ -274,7 +275,7 @@ def run_trajectory(sim=True):
         promotes_outputs=['mission:*'])
 
     traj.link_phases(["climb", "cruise", "descent"], [
-                     "time", Dynamic.Mission.ALTITUDE, Dynamic.Mission.MASS, Dynamic.Mission.DISTANCE], connected=strong_couple)
+                     "time", Dynamic.Mission.MASS, Dynamic.Mission.DISTANCE], connected=strong_couple)
```

Comment:
I don't understand this change. Is it because the altitude is a control with opt=False?

Reply:
Correct; altitude is not varied here